packages/jest-mock/src/index.ts

```diff
@@ -201,6 +201,10 @@ type MockFunctionState<T extends FunctionLike = UnknownFunction> = {
    * List of all the object instances that have been instantiated from the mock.
    */
   instances: Array<ReturnType<T>>;
+  /**
+   * List of all the function contexts that have been applied to calls to the mock.
+   */
+  contexts: Array<unknown>;
```

Comment:
Feels like type could be inferred using `ThisParameterType`. Or? Here is what I was trying out quickly:

```ts
const mockFn = jest.fn<(this: number) => void>();

mockFn.mock.contexts // here the type is `Array<number>`
```

```suggestion
  contexts: Array<ThisParameterType<T>>;
```

Reply:
Wonderful, yes that will work